e2etest/bigint_test.go

```diff
@@ -49,7 +49,7 @@ func TestTransfer_Negative(t *testing.T) {
 	r.NoError(err)
 	balance, err := bc.Factory().Balance(executor)
 	r.NoError(err)
-	r.Equal(0, balance.Cmp(balanceBeforeTransfer))
+	r.Equal(-1, balance.Cmp(balanceBeforeTransfer))
```

Comment:
why are they different? Could you comment? It would be better if we can specify the diffs between these two numbers.

Reply:
fixed unit test bug in latest commit